cmd/manager/main.go

```diff
@@ -75,6 +82,9 @@ var (
 			}
 			if flags.controllerOptions.Namespace == "" {
 				flags.controllerOptions.Namespace = os.Getenv("POD_NAMESPACE")
+				if flags.controllerOptions.Namespace == "" {
```

Comment:
This was intended to be guidance to the developer testing apiserver.  Perhaps it works inside the cluster.  From outside the cluster it was failing with an error message that confused me because the default namespace for _kubectl_ and _istioctl_ is "default" but the default namespace for _manager_ is "".

```
$ KUBECONFIG=~/.kube/config ./manager apiserver --namespace default
I0508 19:25:07.190941   32404 apiserver.go:96] Starting api at :8081
^C
$ KUBECONFIG=~/.kube/config ./manager apiserver --namespace ""
Error: failed to retrieve mesh configuration. an empty namespace may not be set when a resource name is provided
```